src/com/owncloud/android/ui/errorhandling/ErrorMessageAdapter.java

```diff
@@ -443,7 +443,7 @@ private static String getGenericErrorMessageForOperation(RemoteOperation operati
                 ) {
             message = res.getString(R.string.share_link_file_error);
 
-        } else if (operation instanceof UnshareOperation) {
+        } else if (operation instanceof RemoveShareOperation) {
             message = res.getString(R.string.unshare_link_file_error);
```

Comment:
You could rename unshare_link_file_error to remove_link_file_error in order to be consistent with the name of the operation